microservices/first/utils/db_manager.py

```diff
@@ -0,0 +1,134 @@
+import json
+import sqlite3
+from sqlite3 import Error
+from typing import Any, Dict, List, Optional, Union
+
+from . import deployment_status
+
+"""
+Deployment Table Schema:
+id: Integer, Primary key identifying each row
+deployment_id: Integer, Indentifier of deployment
+repo_name: TEXT, Name of repo the deployment is working on
+repo_branch: TEXT, Branch of the repo being worked on
+status: TEXT, (DeploymentStatus in text form) Status of the deployment.
+
+Results Table Schema:
+id: Integer, Primary key identifying each row
+deployment_id: Integer, Foreign Key pointing to Deployment Table "deployment_id". Deployment the result is associated with.
+result: TEXT, one test result object.
+"""
```

Comment:
Please update the FIRST mega with this information

Reply:
This should be put in a markdown documentation file